packages/bsky/src/api/app/bsky/notification/listActivitySubscriptions.ts

```diff
@@ -0,0 +1,123 @@
+import { mapDefined } from '@atproto/common'
+import { AppContext } from '../../../../context'
+import { HydrateCtx, Hydrator } from '../../../../hydration/hydrator'
+import { Server } from '../../../../lexicon'
+import { QueryParams } from '../../../../lexicon/types/app/bsky/notification/listActivitySubscriptions'
+import {
+  HydrationFnInput,
+  PresentationFnInput,
+  RulesFnInput,
+  SkeletonFnInput,
+  createPipeline,
+} from '../../../../pipeline'
+import { uriToDid as didFromUri } from '../../../../util/uris'
+import { Views } from '../../../../views'
+import { clearlyBadCursor, resHeaders } from '../../../util'
+
+export default function (server: Server, ctx: AppContext) {
+  const listActivitySubscriptions = createPipeline(
+    skeleton,
+    hydration,
+    noBlocks,
+    presentation,
+  )
+  server.app.bsky.notification.listActivitySubscriptions({
+    auth: ctx.authVerifier.standard,
+    handler: async ({ params, auth, req }) => {
+      const viewer = auth.credentials.iss
+      const labelers = ctx.reqLabelers(req)
+      const hydrateCtx = await ctx.hydrator.createContext({
+        labelers,
+        viewer,
+      })
+
+      const result = await listActivitySubscriptions(
+        { ...params, hydrateCtx: hydrateCtx.copy({ viewer }) },
```

Comment:
At a glance I don't think the `copy()` is needed here.
```suggestion
        { ...params, hydrateCtx },
```